checkov/terraform/checks/resource/alicloud/K8sEnableNetworkPolicies.py

```diff
@@ -0,0 +1,68 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class K8sEnableNetworkPolicies(BaseResourceCheck):
+    def __init__(self):
+        """
+        Kubernetes Cluster should have Terway or Flannel as CNI Network Plugin as it allows you to use
+        standard Kubernetes network policies
+        https://www.alibabacloud.com/help/en/container-service-for-kubernetes/latest/work-with-terway
+        https://registry.terraform.io/providers/aliyun/alicloud/latest/docs/resources/cs_kubernetes#cluster_network_type
+        The vswitches for the pod network when using Terway.Be careful the pod_vswitch_ids can not equal to
+        worker_vswitch_ids or master_vswitch_ids but must be in same availability zones.
+        Flannel requires pod_cidr.
+        """
+
+        name = "Ensure Kubernetes installs plugin Terway or Flannel to support standard policies"
+        id = "CKV_ALI_26"
+        supported_resources = ['alicloud_cs_kubernetes']
+        categories = [CheckCategories.KUBERNETES]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        # required fields
+        if not (conf.get("pod_vswitch_ids") or conf.get("pod_cidr")):
+            return CheckResult.FAILED
+        # addons
+        if conf.get("addons") and isinstance(conf.get("addons"), list):
+            names = []
+            if len(conf.get("addons")) > 1:
+                addons = conf.get("addons")
+                for addon in addons:
+                    names.append(addon.get("name")[0])
+            else:
+                addons = conf.get("addons")[0]
+                names = addons.get("name")
```

Comment:
```suggestion
            names = [
                addon["name"][0]
                for addon in conf["addons"]
                if addon.get("name")
            ]
```
I think this should do the same